packages/testlab/src/index.ts

```diff
@@ -12,3 +12,4 @@ export * from './test-sandbox';
 export * from './skip-travis';
 export * from './request';
 export * from './http-server-config';
+export * from './misc';
```

Comment:
This is from https://github.com/strongloop/loopback-next/pull/1733, `git` should remove it after #1733 lands and this patch is rebased on top of the new master.